<ui version="4.0" >
 <class>StatisticsForm</class>
 <widget class="QDialog" name="StatisticsForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Statistik</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="font" >
      <font>
       <family>DejaVu Sans</family>
       <pointsize>26</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Statistik</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="font" >
      <font>
       <family>DejaVu Sans</family>
      </font>
     </property>
     <property name="title" >
      <string>Periode</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="font" >
         <font>
          <family>DejaVu Sans</family>
         </font>
        </property>
        <property name="text" >
         <string>Start:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QDateTimeEdit" name="startTime" >
        <property name="displayFormat" >
         <string>yyyy-MM-dd hh:mm</string>
        </property>
        <property name="calendarPopup" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="5" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Slut:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6" >
       <widget class="QDateTimeEdit" name="endTime" >
        <property name="displayFormat" >
         <string>yyyy-MM-dd hh:mm</string>
        </property>
        <property name="calendarPopup" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QCheckBox" name="sinceStart" >
        <property name="font" >
         <font>
          <family>DejaVu Sans</family>
         </font>
        </property>
        <property name="text" >
         <string>Altid</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="font" >
      <font>
       <family>DejaVu Sans</family>
      </font>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Tab 1</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="font" >
      <font>
       <family>DejaVu Sans</family>
      </font>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Close</set>
     </property>
     <property name="centerButtons" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StatisticsForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StatisticsForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
